### PR TITLE
Added config.name param to create workspace params

### PR DESCRIPTION
**Pull request by mlabuda**

### CODE REVIEW

**Comment by mlabuda** [comment]
fixes #115

**Comment by ibuziuk** [comment]
[test]

File: src/main/java/io/fabric8/che/starter/controller/WorkspaceController.java [hunk 1]
```diff
@@ -128,28 +129,20 @@ public WorkspaceLink createOnOpenShift(@RequestParam String masterUrl, @RequestP
...
-            if (wsDescription != null && wsDescription.equals(params.getDescription())) {
-                // Before we can create a project, we must start the new workspace.  First check it's not already running
-                if (!WorkspaceState.RUNNING.toString().equals(ws.getStatus()) && 
-                        !WorkspaceState.STARTING.toString().equals(ws.getStatus())) {
-                    workspaceClient.startWorkspace(cheServerUrl, ws.getId());
-                }
-                return workspaceHelper.getWorkspaceIdeLink(ws);
-            }
+        String workspaceName = params.getWorkspaceName();
+        if (workspaceName != null && !workspaceName.isEmpty()) {
```
Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/controller/WorkspaceController.java]: StringUtil.isBlank

**Comment by ibuziuk** [comment]
@mlabuda great! just one single comment to address

File: src/main/java/io/fabric8/che/starter/controller/WorkspaceController.java (hunk 1 above)
Comment (mlabuda) [inline src/main/java/io/fabric8/che/starter/controller/WorkspaceController.java]: fixed

**Comment by ibuziuk** [comment]
[test]

**Pull request merged**